CHANGELOG.md

```diff
@@ -16,6 +16,14 @@
 
 ---
 
+## v4.5.0
+
+- ⚡️ [RUMF-1115] throttle view updates by addTiming ([#1355](https://github.com/DataDog/browser-sdk/pull/1355))
+- 📝 Update the configuration of the Browser Sessions ([#1322](https://github.com/DataDog/browser-sdk/pull/1322))
+- ✨ [RUMF-1177] add index_in_view to segment meta ([#1342](https://github.com/DataDog/browser-sdk/pull/1342))
+- ⚗ [RUMF-1103] Remove event bridge FF ([#1333](https://github.com/DataDog/browser-sdk/pull/1333))
```

Comment:
If I am not mistaken, this one is actually allowing web view tracking, it could be nice to make that a bit more clear